docker-compose.yml

```diff
@@ -2,8 +2,7 @@ version: "3"
 
 services:
   excalidraw:
-    build: .
-    container_name: excalidraw
+    image: getexcalidraw/excalidraw:latest
```

Comment:
@guillaumebriday I'm not sure about this, the purpose of the `docker-compose` file is for local development. I don't see a reason to use the remote image for this.

Reply:
I will change it